src/plugins/orangehrmAdminPlugin/Api/ModulesAPI.php

```diff
@@ -221,7 +225,11 @@ public function update(): EndpointResourceResult
     {
         $modules = self::CONFIGURABLE_MODULES;
         foreach ($modules as $key => $module) {
```

Comment:
```suggestion
         foreach (self::CONFIGURABLE_MODULES as $key => $module) {
```
:thinking:

Reply:
we have to keep $modules variable, cause we cannot update the content of a constant, self::CONFIGURABLE_MODULES